docs/shop_administration/catalogs.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Catalogs allow creating filtered product lists for special purposes, for example promos, campaigns, or tenders.
+---
+
+# Catalogs
+
+You can create catalogs which contain a selection of products from the system.
+
+Use catalogs, for example, to build special discount catalogs, campaigns, or tenders.
+
+![Catalog menu with a sample catalog](img/catalog.png)
+
+## Creating catalogs
+
+To create a catalog:
+
+1. In the **Product catalog** section in the menu, select **Catalogs**.
+2. Click **Create**.
+3. Enter the name and identifier for the catalog.
+4. Select filters that decide which products are included in the catalog.
+5. Click **Create**.
+
+### Catalog filters
```

Comment:
Maybe it's worth mentioning that after the catalog is created, all available products are displayed and then the user has to filter them out. Since filters constitute the whole idea of the catalog.